src/router.js

```diff
@@ -1,5 +1,69 @@
+import RunModel from './models/run';
+import RunListCollection from './collections/run-list';
+import RunIndexView from './views/posts-index';
+import RunFormView from './views/run-form';
+import RunDetailView from './views/run-detail';
+
 var Router = Backbone.Router.extend({
+  routes: {
+    '': 'listAllRuns',
+    new: 'newRun',
```

Comment:
Check the name of the function below that you want to run when the url is `#new`...